tests/local/network.go

```diff
@@ -91,81 +85,66 @@ func NewLocalNetwork(warpGenesisFile string) *LocalNetwork {
 	_, err = os.Stat(warpGenesisFile)
 	Expect(err).Should(BeNil())
 
-	anrConfig := runner.NewDefaultANRConfig()
-	anrConfig.GlobalCChainConfig = warpEnabledChainConfig
-	manager := runner.NewNetworkManager(anrConfig)
+	subnetA := subnetEvmTestUtils.NewTmpnetSubnet(
+		"A",
+		warpGenesisFile,
```

Comment:
It looks like this was probably existing already, but lets use a different EVM chain ID for each of the subnets that get created here. Otherwise the local chains will be subject to possible replayed transactions or transactions accidentally being sent to the wrong chain being valid.

It'd probably be easiest to make the `warpGenesisFile` a template with a <EVM_CHAIN_ID>, and provide a util for creating a temp file with a given chain ID value.

Reply:
addressed in https://github.com/ava-labs/teleporter/pull/401/commits/32f349e1a4eb65f5c0491f16bd69bc2fbaac34e9